### ISSUE TITLE
 Bug:  Confused with nuitka

### ISSUE THREAD

**Issue by testingpurposessza** [Smug246/Luna-Grabber#735]
### What Did You Expect To Happen?

 it says If you want to use Nuitka, you need to select a version 3.4.X - 3.11.X
Enter the number of the desired Python version from the list above: but if i put any in it just exits it

### What Happened?

 i downloaded this new versions and then this happened

### Add Screenshots

![image](https://github.com/Smug246/Luna-Grabber/assets/166067998/768d7ca8-6b5a-4899-8b30-0e5703b86117)

### Which Options Did You Select?

Discord Info, Fake Error, Wifi Info, Ping, Anti Debug/VM, Minecraft Info, System Info, Browser Info, Roblox Info, Anti Spam

### Check the following before submitting (If Applicable)

- [X] I have checked the existing [issues](https://github.com/Smug246/Luna-Grabber/issues)
- [X] I have followed the [installation](https://github.com/Smug246/Luna-Grabber?tab=readme-ov-file#installation) steps
- [X] I have attempted to resolve my problem by myself

**Comment by Smug246** [comment]
you see the number at the start. you just type that. so in your case number 1

**Comment by Serpensin** [comment]
Exactly. I set it that way, because in theory, you could've installed multiple pythons with the same version.

**Comment by testingpurposessza** [comment]
> > Exactly. I set it that way, because in theory, you could've installed multiple pythons with the same version.
> 
> we may need to word it differently to make it even simpler lol

yeah ma brain stopped workin im try it rn ty

**Comment by testingpurposessza** [comment]
> > Exactly. I set it that way, because in theory, you could've installed multiple pythons with the same version.
> 
> we may need to word it differently to make it even simpler lol

worked ty man

**Comment by testingpurposessza** [comment]
> > Exactly. I set it that way, because in theory, you could've installed multiple pythons with the same version.
> 
> we may need to word it differently to make it even simpler lol

i tested it on myself and it seemed to not send it to the discord server * i dont plan on using against ppl i dont know *
